spec/controllers/concerns/laying_date_helper_spec.rb

```diff
@@ -3,50 +3,61 @@
 RSpec.describe LayingDateHelper, type: :helper do
   describe '#get_date' do
 
-    let(:work_package) { double('work_package', work_packaged_thing: work_packaged_thing) }
+    let(:laying_connected_thing) { double('laying_connected_thing', work_packaged_thing: work_packaged_thing, is_a?: true) }
```

Comment:
Could you be specific here and get is_a? to respond with Parliament::Grom::Decorator::WorkPackage?

Reply:
🎉